inc/class-s3-uploads.php

```diff
@@ -277,4 +348,142 @@ public function copy_image_from_s3( $file ) {
 		copy( $file, $temp_filename );
 		return $temp_filename;
 	}
+
+	/**
+	 * Check if the attachment is private.
+	 *
+	 * @param integer $attachment_id
+	 * @return boolean
+	 */
+	public function is_private_attachment( int $attachment_id ) : bool {
+		return apply_filters( 's3_uploads_is_attachment_private', true, $attachment_id );
```

Comment:
@joehoyle I think this should be:
```
return apply_filters( 's3_uploads_is_attachment_private', false, $attachment_id );
```

Instead of forcing private attachments on everyone by default, we should allow them to opt in via:
```
 add_filter( 's3_uploads_is_attachment_private', '__return_true' );
```

Unless the intended use case you have is for it to be on by default. If that's the case should we update the above filter to in the readme:
```
 add_filter( 's3_uploads_is_attachment_private', '__return_false' );
```